Add HUNGER_LOGIN_URL to take precedence over LOGIN_URL

stenius opened the pull request:
I was using this var for something else already and I wanted to do some different stuff for pages that were protected by hunger.  There's not a better way to do this right?

stenius committed:
check for a new setting specific to hunger if it exiists

yesimon commented:
This is a good idea. I changed the title. If you can add a test to check for proper override and set the default of `HUNGER_LOGIN_URL` to `LOGIN_URL` in `hunger/util.py` then I can merge it properly.

stenius committed:
check HUNGER_LOGIN_URL overrides LOGIN_URL

stenius commented:
Thanks for getting this in there Simon,  I added a note to the docs about the variable too.

wooyek committed:
Update URL to resolve RemovedInDjango110Warning

RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got hunger.views.verify_invite). Pass the callable instead.
  name='hunger-verify'),

RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  name='hunger-invalid'),

stenius committed:
remove url.patterns import for django 1.10

stenius committed:
Merge pull request #1 from stenius/feature/RemovedInDjango110Warning

Feature/removed in django110 warning

stenius committed:
updates

stenius committed:
Update urls.py

stenius committed:
fix issue with 404s, 500s, and redirects
